src/pepr/operator/reconcilers/index.ts

```diff
@@ -47,7 +51,7 @@ export function shouldSkip(cr: UDSPackage) {
  * @param status The new status
  */
 export async function updateStatus(cr: UDSPackage, status: PkgStatus) {
-  Log.debug(cr.metadata, `Updating status to ${status.phase}`);
+  log.debug(cr.metadata, `Updating status to ${status.phase}`);
```

Comment:
I think here we can/should find a way to clean up this log (it looks a bit messy currently logging out the full metadata). We can probably just log out the name/namespace?